packages/synapse-react-client/src/style/components/entity_finder/_entity-finder.scss

```diff
@@ -32,22 +31,24 @@ $-splitter-width: 12px;
   width: 100%;
 }
 
+.MuiDialog-root {
+  .EntityFinder {
+    &__MainPanel.MainPanelDualPane,
+    &__MainPanel.MainPanelSearch {
+      // Hack to make the dialog responsive to the width of the entity finder
+      // We may be able to remove this if we remove/replace the EntityFinder's `AutoSizer` component with something other
+      // than the 'react-base-table' implementation
+      width: 85vw;
+    }
+  }
+}
+
 .EntityFinder {
+  --entity-finder-height: #{$-finder-height};
+  --entity-tree-header-height: 65px;
```

Comment:
Fix a calculation issue in the experimental EntityFileBrowser (where `--entity-tree-header-height` should be 0px) by creating CSS variables with these values.